data/nginx.conf

```diff
@@ -14,6 +14,10 @@
      
      include /etc/nginx/mime.types;
 
+     # Block access to sensitive files and return 404 to make it indistinguishable from a missing file
+     location ~ (\.htaccess$|\.htpasswd$|\.ini$|\.log$|\.sh$|\.inc$|\.bak$|\.twig$|\.sql$) {
```

Comment:
.htaccess and .htpassword is allready blocked by 
https://github.com/FOSSBilling/FOSSBilling/blob/e0cb529ca86351cfa80e1a34a563853d81a092eb/data/nginx.conf#L56-L58

How ever I suggest blocking  all hidden files by:

https://github.com/hestiacp/hestiacp/blob/e3e172d893f20c303941a1ab2d846eeb823a9179/install/deb/templates/web/nginx/php-fpm/default.tpl#L42-L45

Also this looks nicer:
 location ~* ^.+\.(ini|sh|inc|bak|twig|sql|)$ {